msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Meeting app extensibility
+author: laujan
+description: Understand the meeting app extensibility 
+ms.topic: conceptual
+---
+
+# Meeting app extensibility
+
+Teams’ meeting app extensibility is based on the following concepts:
+
+✔ Meeting lifecycle has stages such as before, during, and after meeting time frame.  
+✔ Participant roles in a meeting such as meeting organizer, presenter, or attendee.  
+✔ User types in a meeting such as in-tenant, guest, federated, or anonymous Teams user.
```

Comment:
There are various [user types](https://docs.microsoft.com/en-us/microsoftteams/non-standard-users#:~:text=An%20anonymous%20user%20is%20a,their%20Microsoft%20or%20organization's%20account.) in a meeting; In-tenant, [Guest](https://docs.microsoft.com/en-us/microsoftteams/guest-access), [Federated](https://docs.microsoft.com/en-us/microsoftteams/manage-external-access), and Anonymous users